[2025-06-24T20:10:33.872+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: northwind.carregamento_postgres manual__2025-06-24T20:04:13.108195+00:00 [queued]>
[2025-06-24T20:10:33.890+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: northwind.carregamento_postgres manual__2025-06-24T20:04:13.108195+00:00 [queued]>
[2025-06-24T20:10:33.891+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-06-24T20:10:33.918+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): carregamento_postgres> on 2025-06-24 20:04:13.108195+00:00
[2025-06-24T20:10:33.923+0000] {standard_task_runner.py:60} INFO - Started process 243 to run task
[2025-06-24T20:10:33.927+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'northwind', 'carregamento_postgres', 'manual__2025-06-24T20:04:13.108195+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpvvhmdpcu']
[2025-06-24T20:10:33.929+0000] {standard_task_runner.py:88} INFO - Job 98: Subtask carregamento_postgres
[2025-06-24T20:10:33.991+0000] {task_command.py:423} INFO - Running <TaskInstance: northwind.carregamento_postgres manual__2025-06-24T20:04:13.108195+00:00 [running]> on host 2f206bf85820
[2025-06-24T20:10:34.112+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lrbf@cin.ufpe.br' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='northwind' AIRFLOW_CTX_TASK_ID='carregamento_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-24T20:04:13.108195+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-24T20:04:13.108195+00:00'
[2025-06-24T20:10:34.874+0000] {docker.py:359} INFO - Starting docker container from image meltano:latest
[2025-06-24T20:10:36.702+0000] {docker.py:429} INFO - Running meltano install...
[2025-06-24T20:10:40.654+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.653715Z[0m [[32m[1minfo     [0m] [1mInstalling 7 plugins          [0m
[2025-06-24T20:10:40.664+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.663407Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-postgres'[0m
[2025-06-24T20:10:40.678+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.677510Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-csv'[0m
[2025-06-24T20:10:40.694+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.693870Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-csv-latest'[0m
[2025-06-24T20:10:40.708+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.707326Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-postgres'[0m
[2025-06-24T20:10:40.733+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.732698Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv'[0m
[2025-06-24T20:10:40.758+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.757933Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv-postgres'[0m
[2025-06-24T20:10:40.787+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:40.786120Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv-csv'[0m
[2025-06-24T20:10:43.202+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:43.191400Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-postgres'[0m
[2025-06-24T20:10:43.209+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:43.199754Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-postgres'[0m
[2025-06-24T20:10:50.245+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:50.240665Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-csv' [0m
[2025-06-24T20:10:50.369+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:50.368573Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-csv-latest'[0m
[2025-06-24T20:10:50.966+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:50.942778Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv' [0m
[2025-06-24T20:10:51.588+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:51.587681Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv-postgres'[0m
[2025-06-24T20:10:51.669+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:51.668266Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv-csv'[0m
[2025-06-24T20:10:51.678+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:51.677297Z[0m [[32m[1minfo     [0m] [1mInstalled 7/7 plugins         [0m
[2025-06-24T20:10:53.168+0000] {docker.py:429} INFO - Meltano install completed. Executing command: meltano el tap-csv-latest target-postgres
[2025-06-24T20:10:54.543+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:54.542856Z[0m [[32m[1minfo     [0m] [1mEnvironment 'dev' is active   [0m
[2025-06-24T20:10:55.021+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:55.019797Z[0m [[32m[1minfo     [0m] [1mRunning extract & load...     [0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m
[2025-06-24T20:10:56.101+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.100229Z[0m [[32m[1minfo     [0m] [1mTraceback (most recent call last):[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.101+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.100620Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/bin/tap-csv", line 8, in <module>[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.102+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.100870Z[0m [[32m[1minfo     [0m] [1m    sys.exit(TapCSV.cli())    [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.102+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.101157Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/click/core.py", line 1442, in __call__[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.102+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.101594Z[0m [[32m[1minfo     [0m] [1m    return self.main(*args, **kwargs)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.103+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.101925Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/click/core.py", line 1363, in main[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.103+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.102222Z[0m [[32m[1minfo     [0m] [1m    rv = self.invoke(ctx)     [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.103+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.102448Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/plugin_base.py", line 92, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.104+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.102687Z[0m [[32m[1minfo     [0m] [1m    return super().invoke(ctx)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.104+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.102919Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/click/core.py", line 1226, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.104+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.103213Z[0m [[32m[1minfo     [0m] [1m    return ctx.invoke(self.callback, **ctx.params)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.105+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.103650Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/click/core.py", line 794, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.105+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.104386Z[0m [[32m[1minfo     [0m] [1m    return callback(*args, **kwargs)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.105+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.105011Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 516, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.105+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.105320Z[0m [[32m[1minfo     [0m] [1m    tap = cls(                [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.106+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.105691Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 118, in __init__[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.108+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.107556Z[0m [[32m[1minfo     [0m] [1m    self.setup_mapper()       [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.111+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.108532Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 194, in setup_mapper[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.112+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.111693Z[0m [[32m[1minfo     [0m] [1m    self.mapper.register_raw_streams_from_catalog(self.catalog)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.112+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.112064Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 187, in catalog[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.113+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.112347Z[0m [[32m[1minfo     [0m] [1m    self._catalog = self.input_catalog or self._singer_catalog[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.113+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.112842Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 340, in _singer_catalog[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.113+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.113140Z[0m [[32m[1minfo     [0m] [1m    for stream in self.streams.values()[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.113+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.113386Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 149, in streams[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.114+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.113814Z[0m [[32m[1minfo     [0m] [1m    for stream in self.load_streams():[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.114+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.114170Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/singer_sdk/tap_base.py", line 373, in load_streams[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.114+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.114434Z[0m [[32m[1minfo     [0m] [1m    for stream in self.discover_streams():[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.115+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.114760Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/tap_csv/tap.py", line 89, in discover_streams[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.115+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.115304Z[0m [[32m[1minfo     [0m] [1m    return [                  [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.116+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.115595Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.10/site-packages/tap_csv/tap.py", line 92, in <listcomp>[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.116+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.115940Z[0m [[32m[1minfo     [0m] [1m    name=file_config.get("entity"),[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.117+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.116284Z[0m [[32m[1minfo     [0m] [1mAttributeError: 'str' object has no attribute 'get'[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.347+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.346112Z[0m [[32m[1minfo     [0m] [1mtarget-postgres v0.4.3, Meltano SDK v0.47.2[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2m2025-06-24T20:10:56.346505Z[0m [[32m[1minfo     [0m] [1mSkipping parse of env var settings...[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.347+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.346719Z[0m [[32m[1minfo     [0m] [1mTraceback (most recent call last):[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.348+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.346944Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/bin/target-postgres", line 8, in <module>[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.348+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.347148Z[0m [[32m[1minfo     [0m] [1m    sys.exit(TargetPostgres.cli())[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.348+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.347310Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 1442, in __call__[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.349+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.347581Z[0m [[32m[1minfo     [0m] [1m    return self.main(*args, **kwargs)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.349+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.347763Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 1363, in main[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.349+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.348088Z[0m [[32m[1minfo     [0m] [1m    rv = self.invoke(ctx)     [0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.350+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.348301Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/singer_sdk/plugin_base.py", line 97, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.350+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.348540Z[0m [[32m[1minfo     [0m] [1m    return super().invoke(ctx)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.350+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.348760Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 1226, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.350+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.349026Z[0m [[32m[1minfo     [0m] [1m    return ctx.invoke(self.callback, **ctx.params)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.351+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.349271Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/click/core.py", line 794, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.351+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.349641Z[0m [[32m[1minfo     [0m] [1m    return callback(*args, **kwargs)[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.351+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.349949Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/singer_sdk/target_base.py", line 583, in invoke[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.351+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.350411Z[0m [[32m[1minfo     [0m] [1m    target = cls(             [0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.352+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.350924Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.10/site-packages/target_postgres/target.py", line 67, in __init__[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.352+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.351214Z[0m [[32m[1minfo     [0m] [1m    and self.config.get("dialect+driver") is not None[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.352+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.351471Z[0m [[32m[1minfo     [0m] [1mAssertionError: Need either the sqlalchemy_url to be set or host, port, user, password, and dialect+driver to be set[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.463+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.462090Z[0m [[31m[1merror    [0m] [1mExtraction failed             [0m [36mcode[0m=[35m1[0m [36mmessage[0m=[35m"AttributeError: 'str' object has no attribute 'get'"[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m
[2m2025-06-24T20:10:56.462361Z[0m [[31m[1merror    [0m] [1mLoading failed                [0m [36mcode[0m=[35m1[0m [36mmessage[0m=[35m'AssertionError: Need either the sqlalchemy_url to be set or host, port, user, password, and dialect+driver to be set'[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m
[2025-06-24T20:10:56.464+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.462709Z[0m [[32m[1minfo     [0m] [1mELT could not be completed: Extractor and loader failed.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.464+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.462978Z[0m [[32m[1minfo     [0m] [1mFor more detailed log messages re-run the command using 'meltano --log-level=debug ...' CLI flag.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.464+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.463190Z[0m [[32m[1minfo     [0m] [1mNote that you can also check the generated log file at '/opt/meltano/.meltano/logs/elt/2025-06-24T201054--tap-csv-latest--target-postgres/b0bf94be-6ae3-477e-97a3-57c1c526b429/elt.log'.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.465+0000] {docker.py:429} INFO - [2m2025-06-24T20:10:56.463420Z[0m [[32m[1minfo     [0m] [1mFor more information on debugging and logging: https://docs.meltano.com/reference/command-line-interface#debugging[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35mb0bf94be-6ae3-477e-97a3-57c1c526b429[0m [36mstate_id[0m=[35m2025-06-24T201054--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-06-24T20:10:56.481+0000] {docker.py:429} INFO - Need help fixing this problem? Visit http://melta.no/ for troubleshooting steps, or to
join our friendly Slack community.

ELT could not be completed: Extractor and loader failed.
For more detailed log messages re-run the command using 'meltano --log-level=debug ...' CLI flag.
Note that you can also check the generated log file at '/opt/meltano/.meltano/logs/elt/2025-06-24T201054--tap-csv-latest--target-postgres/b0bf94be-6ae3-477e-97a3-57c1c526b429/elt.log'.
For more information on debugging and logging: https://docs.meltano.com/reference/command-line-interface#debugging
[2025-06-24T20:10:57.986+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 376, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2025-06-24T20:10:58.000+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=northwind, task_id=carregamento_postgres, execution_date=20250624T200413, start_date=20250624T201033, end_date=20250624T201057
[2025-06-24T20:10:58.145+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/email.py:154 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2025-06-24T20:10:58.147+0000] {configuration.py:1046} WARNING - section/key [smtp/smtp_user] not found in config
[2025-06-24T20:10:58.148+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-06-24T20:10:58.175+0000] {configuration.py:1046} WARNING - section/key [smtp/smtp_user] not found in config
[2025-06-24T20:10:58.175+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-06-24T20:10:58.177+0000] {taskinstance.py:826} ERROR - Failed to send email to: ['lrbf@cin.ufpe.br']
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2334, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2499, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2516, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 376, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1000, in _email_alert
    send_email(task.email, subject, html_content)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 80, in send_email
    return backend(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 154, in send_email_smtp
    send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 272, in send_mime_email
    smtp_conn = _get_smtp_connection(smtp_host, smtp_port, smtp_timeout, smtp_ssl)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 316, in _get_smtp_connection
    return smtplib.SMTP(host=host, port=port, timeout=timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 255, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.10/smtplib.py", line 341, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 312, in _get_socket
    return socket.create_connection((host, port), timeout,
  File "/usr/local/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/local/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 824, in _handle_failure
    task_instance.email_alert(error, failure_context["task"])
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2946, in email_alert
    _email_alert(task_instance=self, exception=exception, task=task)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1002, in _email_alert
    send_email(task.email, subject, html_content_err)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 80, in send_email
    return backend(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 154, in send_email_smtp
    send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 272, in send_mime_email
    smtp_conn = _get_smtp_connection(smtp_host, smtp_port, smtp_timeout, smtp_ssl)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 316, in _get_smtp_connection
    return smtplib.SMTP(host=host, port=port, timeout=timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 255, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.10/smtplib.py", line 341, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 312, in _get_socket
    return socket.create_connection((host, port), timeout,
  File "/usr/local/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/local/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
[2025-06-24T20:10:58.211+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 98 for task carregamento_postgres (Docker container failed: {'StatusCode': 1}; 243)
[2025-06-24T20:10:58.271+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-24T20:10:58.312+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
